cmake/Cuda.cmake

```diff
@@ -267,7 +269,9 @@ list(APPEND mshadow_LINKER_LIBS ${CUDA_CUDART_LIBRARY}
 if(CUDA_ARCH_ALL)
   set(mshadow_known_gpu_archs "${CUDA_ARCH_ALL}")
 else()
-  if(${CUDA_VERSION} EQUAL 9.0 OR ${CUDA_VERSION} GREATER 9.0)
+  if(${CUDA_VERSION} EQUAL 10.0 OR ${CUDA_VERSION} GREATER 10.0)
+    set(mshadow_known_gpu_archs "30 35 50 52 60 61 70 75")
```

Comment:
In Makefile, the cuda archs simply include a complete list, and at build time it invokes nvcc to check whether an arch is supported. Such an approach should spare us the need to update this section every time.
https://github.com/apache/incubator-mxnet/blob/master/Makefile#L345-L349

Reply:
I wonder if there's a way of invoking nvcc in such a way that it returns us a parsable list of existing compute capabilities.